modules/util/css-builder/src/main/resources/META-INF/maven/com.liferay/com.liferay.css.builder/plugin-help.xml

```diff
@@ -49,7 +49,7 @@
 				<parameter>
 					<name>docrootDirName</name>
 					<type>java.lang.String</type>
-					<deprecated>As of 2.1.0, replaced by {@link #setBaseDir(File)}</deprecated>
+					<deprecated>As of Judson (7.1.x), replaced by {@link #setBaseDir(File)}</deprecated>
```

Comment:
@petershin, can you please try to figure out why this change happened? I think there's something wrong with SF or with the Javadoc Formatter 🤔 Also, can you revert these wrong changes made in `modules/sdk` and `modules/util`? Thanks!

Reply:
@Ithildir - I think its with the `Javadoc Formatter` will take a look

Reply:
I'm reverting them, if you want to work on something else in the meantime 😊

Reply:
Appreciate it. Will send a patch for this issue